src/api/app/views/webui/obs_factory/distributions/show.html.haml

```diff
@@ -87,7 +87,7 @@
 
     %h2
       %i.fa.fa-2.fa-wrench
-      = link_to "openQA results for #{@versions[:totest]}", "#{openqa_links_helper}/tests/overview?distri=opensuse&version=#{@distribution.openqa_version}&build=#{@versions[:totest]}&group=#{@distribution.openqa_group}"
+      = link_to "openQA results for #{@versions[:totest]}", "#{openqa_links_helper}/tests/overview?distri=opensuse&version=#{@distribution.openqa_version}&build=#{@versions[:totest]}"
```

Comment:
And probably a feature test for this one. Though this could also go to a helper, which would be easier to test and allows to reduce the duplication.

If you want we can take care of this. Here would be an example for a helper test if you want to give it a try: https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/helpers/webui/webui_helper_spec.rb#L6

Reply:
I would appreciate greatly if you pick it up and show me how it's done properly so that for my next PR I know how to do it properly

Reply:
Okay. I'll try to provide some tests today